assignment.rb

```diff
@@ -0,0 +1,103 @@
+# Accesses the csv file with user info
+require "csv"
+
+#Creates a User class with appropriate methods
+class User
+  attr_accessor :name, :pin
+  
+  def initialize(name, pin, balance)
+    @name = name
+    @pin = pin
+    @balance = balance
+  end
+
+  def name
+    @name
+  end
+
+  def pin
+    @pin
+  end
+
+  def balance
+    @balance.to_i
+  end
+
+  def check_balance
+    puts "Your current balance is #{balance}"
```

Comment:
You'll probably want methods like this to return some sort of value and then do the `puts`ing elsewhere